var/spack/repos/builtin/packages/nim/httpclient_ssl_default-1.4.patch

```diff
@@ -0,0 +1,185 @@
+From c7d090c418f7847bdfa1fd3bfed831470665c3e9 Mon Sep 17 00:00:00 2001
```

Comment:
Is this a patch from https://github.com/nim-lang/Nim/commit/abac35e7437dd1ac2b3687dfa51de7f9d4b6e853?  If so, could it be retrieved using the url?  (See an example at starting at https://github.com/spack/spack/blob/dd8dff7872056161f9320a3b8c0329fbbef69d35/var/spack/repos/builtin/packages/dealii/package.py#L264.)

Reply:
Upstreamed patches are preferred over patch files when possible. We already have issues with using too many inodes at some facilities. For more on patches, see https://spack.readthedocs.io/en/latest/packaging_guide.html#patch.

Reply:
Done, thanks.